queries/c_sharp/highlights.scm

```diff
@@ -114,8 +140,7 @@
   (identifier) @type)
 
 (using_directive
```

Comment:
Can `(using_directive)` identifiers (and ones in `(qualified_name)`) be `@module` instead of `@type`? Also is it intended that namespace declarations like `namespace Foo.Bar.Baz`, only `Baz` is given just module while `Foo` and `Bar` are given `@module` overridden by `@type`?

Reply:
using ones can't be determined without semantic info, so no point trying there - but the namespace ones could probably be all `@module`, it's just not easy due to how the children are structured but we could use has-ancestor here since there won't be many anyways

Reply:
ah man. maybe who cares then :laughing: I had hoped it would be an easy change